modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -138,6 +138,7 @@
 
 .liferay-editable {
 	overflow: auto;
+	position: inherit !important;
```

Comment:
Without this "fix", the combo of text formatting styles will be render like this:

![image](https://user-images.githubusercontent.com/7663513/143496728-dbe3f444-de1f-4edb-bde5-48c260cd7054.png)

It is caused because somewhere is defining `style="position: relative;"` to the main CKEditor element but I'm not finding where. 

My investigations concluded that:

1. It don't came from server side. I saw the received markup on network tab
2. I searched for CKEditor APIs that may be adding this style but I don't found
3. I searched for something related to the skin JS file but it is not adding anything
4. This error is reproduced in ckeditor-sample-web module, so, isn't anything related to any side-effect made by a JavaScript file from a module.

Reply:
We are avoiding usage of `!important` just for this case. It's the only way to override third-party inline styles :+1:

Reply:
> We are avoiding usage of !important just for this case

Do you have an alternative for this? 😓

Reply:
or, Do you have any idea where this `style="position: relative;"` may be added to CKEditor element?

Reply:
If we need to override styles, there is no better alternative. The only better solution is remove what's placing the style, if it's under our control.

Reply:
> The only better solution is remove what's placing the style, if it's under our control.

You have no idea how I searched for the LOC that is introducing this side effect. However, I'm not finding. Do you have any clue to add in my investigation? I added https://github.com/liferay-frontend/liferay-portal/pull/1663#discussion_r757114238